### PR TITLE
Products cards

### PR DESCRIPTION
**Pull request by Manju-sree**
### ProductCards in Products page
I have added the ProductCards and static FilterBar component in Products page. It's still in the experimental phase. So ignore other files. Focus only on the below mentioned files. 

## Files to check
please check all jsx files.

## Files to Ignore:
please ignore all .css files, as they are reviewed from previous project repo.

Preview : https://deploy-preview-5--angry-dubinsky-3738ba.netlify.app/Products

### CODE REVIEW

File: src/components/FilterBar/FilterBar.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    <li><label for="radio-button" className="form-label">
+                        <input type="radio" className="form-checkbox-field" name="sort" />
+                        2 Stars & above
+                    </label>
+                    </li>
+                    <li><label for="radio-button" className="form-label">
+                        <input type="radio" className="form-checkbox-field" name="sort" />
+                        1 Stars & above
+                    </label>
+                    </li>
```
Comment (JantuDeb) [inline src/components/FilterBar/FilterBar.jsx]: You can use a ```[4,3,2,1].map(num=><li></li>)```

File: src/components/FilterBar/FilterBar.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    <li><label className="form-label">
+                        <input type="checkbox" className="form-checkbox-field"/>
+                            Concealer-brushes
+                    </label>
+                    </li>
+                    <li><label className="form-label">
+                        <input type="checkbox" className="form-checkbox-field"/>
+                            Blush-brushes
+                    </label>
```
Comment (JantuDeb) [inline src/components/FilterBar/FilterBar.jsx]:
```
const categories = ["Foundation-brushes", "Eyeliner-brushes"]
```
You can use a ```categories.map(num=><li></li>)```
In this way you will not duplicate `html`

**Review by JantuDeb — APPROVED** [review]
Added a suggestion

File: src/components/FilterBar/FilterBar.jsx (hunk 2 above)
Comment (Manju-sree) [reply src/components/FilterBar/FilterBar.jsx]: done changes. thanks for suggestions

File: src/components/FilterBar/FilterBar.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                        <input type="checkbox" className="form-checkbox-field" />
+                        {categoryName}
+                    </label></li>)}
+                    <div className="filter-divider"></div>
+                    <li className="filter-subtitle">Ratings</li>
+                    {[4, 3, 2, 1].map(num => <li><label for="radio-button" className="form-label">
+                        <input type="radio" className="form-checkbox-field" name="sort" />
+                        {num} Stars & above
+                    </label></li>)}
+                </ul>
```
Comment (mariyasada) [inline src/components/FilterBar/FilterBar.jsx:40]: liked that you are implemented categories and ratings like this.

**Review by mariyasada — APPROVED** [review]
the code structure is good. why do you set brand as an empty string in product data

**Comment by Manju-sree** [comment]
> the code structure is good. why do you set brand as an empty string in product data

Now time being it is empty, we will implement it later

**Pull request merged**